advanced/Scripts/piholeDebug.sh

```diff
@@ -330,8 +330,7 @@ function dumpPiHoleLog {
 
 # Anything to be done after capturing of pihole.log terminates
 function finalWork {
-	echo "::: Finshed debugging!" 
-	echo "::: Debug log can be found at : /var/log/pihole_debug.log"
+	echo "::: Debug log can be found at : $(cat /var/log/pihole_debug.log | curl --silent -F 'sprunge=<-' http://sprunge.us)"
```

Comment:
What about some sort of check for upload success prior and printing this message.  If sprunge.us were down or DNS were broke then fallback to the old behavior would be preferable so the user is not left confused.